src/nnet3bin/nnet3-copy-egs.cc

```diff
@@ -301,7 +125,16 @@ int main(int argc, char *argv[]) {
                 "feature left-context that we output.");
     po.Register("right-context", &right_context, "Can be used to truncate the "
                 "feature right-context that we output.");
-
+    po.Register("weights", &weight_str,
```

Comment:
cancel that previous comment, but can you please register these options with better strings?
E.g. specify that it's the Rspecifier of a table indexed by the key of the input examples.